src/common.cc

```diff
@@ -213,7 +216,7 @@ void SizeMap::Init() {
   for (size_t size = kAlignment; size <= kMaxSize; size += AlignmentForSize(size)) {
     CHECK_CONDITION((size % AlignmentForSize(size)) == 0);
 
-    int blocks_to_move = NumMoveSize(size) / 4;
+    int blocks_to_move = NumMoveSize(size) / kObjectsPerBlock;
```

Comment:
MDB's changes on v2.5 are a little different from this.
For the numerator, we don't use NumMoveSize(size), we use kTargetTransferBytes.

The difference is that NumMoveSize is clamped to `[2, FLAGS_tcmalloc_transfer_num_objects]`.
The default (which we don't override at runtime) for that upper bound, known as `kDefaultTransferNumObjecs`, is 512 in gperftools-2.5. In gperftools-2.7 that default was changed from 512 to 32. I'm analyzing the consequences of that upstream change. I hope it makes the NumMoveSize/4 vs kTargetTransferBytes issue irrelevant.

(our customization to this line of the vendored gperftools-2.5)
https://github.com/mongodb/mongo/commit/99436618a0fe429ae5519b02e13f0b22cac5eba1#diff-d1ee3ce5ce38cc5968018fec312d8910R144

(gperftools-2.5's `kDefaultTransferNumObjecs` of 512)
https://github.com/mongodb/mongo/commit/99436618a0fe429ae5519b02e13f0b22cac5eba1#diff-d1ee3ce5ce38cc5968018fec312d8910R47